index.html

```diff
@@ -4,191 +4,272 @@
   <head>
     <link rel="stylesheet" href ="normalize.css">
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Static Comp 🎨</title>
   </head>
   <body>
-    <header>
+    <header class="main-header">
       <div class="logo-admin">
         <div class="admin-dropdown">
           <img class="admin-icon" src="./assets/icons8-user-50 (2).png" alt="admin-icon">
           <p class="admin-user">Admin Username</p>
           <img class="drop-down" src="./assets/icons8-triangle-arrow-48.png" alt="menu-drop-down">
         </div>
-        <div>
+        <div class="branding">
           <h21 class="company-name">2Paint!</h2>
-          <img class="paint-logo" src="./assets/paintbrush.png" alt="paintbrush">
+          <img class="paint-logo" src="./assets/icons8-paint-50.png" alt="paintbrush">
         </div>
       </div>
       <nav>
```

Comment:
I am curious about your decision to have a nav bar and then nested sections. Because each item has a class name assigned to it, there's the potential that the added sections are redundant.